Tabloid/client/src/components/posts/PostList.js

```diff
@@ -15,10 +17,17 @@ const PostList = () => {
 
   return (
     <div className="container">
-      <div className="row">
-        {posts.map((post) => (
-          <Post post={post} key={post.id} />
-        ))}
+      <div className="row justify-content-center">
+        <ListGroup>
+          {posts.map((post) => {
+            return (
+              <ListGroupItem key={post.id}>
+                <Post post={post} key={post.id} />
```

Comment:
If there is a key attribute on the parent ListGroupItem component, is a key attribute also required on the Post component?

Reply:
I've consolidated the key attribute to just the ListGroupItem and I also added an alt attribute to the post header image's html img tag. I was getting a compiler warning about not having done so.